protocol/daemons/server/server.go

```diff
@@ -47,60 +40,14 @@ func NewServer(
 		gsrv:          grpcServer,
 		fileHandler:   fileHandler,
 		socketAddress: socketAddress,
-		updateMonitor: types.NewUpdateFrequencyMonitor(types.DaemonStartupGracePeriod, logger),
 	}
 }
 
 // Stop stops the daemon server's gRPC service.
 func (server *Server) Stop() {
-	server.updateMonitor.Stop()
 	server.gsrv.Stop()
 }
 
-// DisableUpdateMonitoringForTesting disables the update monitor for testing purposes. This is needed in integration
```

Comment:
Removed health monitor from daemon server and attached it to the app as a top level field, so this pass-through interface is no longer necessary.